js/client-chat-tournament.js

```diff
@@ -355,10 +355,16 @@
 						this.info.isActive = true;
 					}
 
-					if ('format' in this.updates || 'isJoined' in this.updates) {
+					if ('format' in this.updates) {
 						this.$format.text(window.BattleFormats && BattleFormats[this.info.format] ? BattleFormats[this.info.format].name : this.info.format);
 						this.updateTeams();
 					}
+					if ('customFormat' in this.updates) {
```

Comment:
Why not just send the custom format name in the `format` field? You seem to be handling them roughly the same way.